src/TopDownProteomics/Chemistry/ChemicalFormula.cs

```diff
@@ -28,14 +28,28 @@ public ChemicalFormula(IEnumerable<IEntityCardinality<IElement>> elements)
             _elements = elements.ToList();
         }
 
+        ///// <summary>
+        ///// Adds the element.
+        ///// </summary>
+        ///// <param name="element">The element.</param>
+        ///// <param name="count">The count.</param>
+        //public void AddElement(IElement element, int count)
+        //{
+        //    _elements.Add(new EntityCardinality<IElement>(element, count));
+        //}
+
         /// <summary>
-        /// Adds the element.
+        /// Waters the specified element provider.
```

Comment:
I suppose it is spring, but I didn't know we had to water element providers. 😄

Reply:
Ha, I saw that as I was committing and thought it was too funny to fix! Someone should throw that in their next PR ...